src/components/common/modal/ModalActionsByFeedType.js

```diff
@@ -16,6 +16,9 @@ const log = logger.child({ from: 'ModalActionsByFeed' })
 const ModalActionsByFeedType = ({ theme, styles, item, handleModalClose }) => {
   const [showErrorDialog] = useErrorDialog()
   const [state, setState] = useState({})
+  const [currentUserName, setCurrentUserName] = useState('')
+
+  userStorage.getProfileFieldValue('fullName').then(setCurrentUserName)
```

Comment:
profile details always exists in the simplestore or gdstore so we dont need to read it async from userstorage but we can read it sync from the store.

Reply:
@sirpy I tried to change this functionality like that: https://prnt.sc/p9w0io also wrap this component with GDStore.Container
Bu the profile is always an empty object, meanwhile, it is working fine for the rest places where the gdStore used.
Did not push the changes as it is not working yet.

Reply:
@yaroslav-fedyshyn-nordwhale yeah you are right gdstore is not available to modals.
i've committed to this branch some changes to make it available, try now

Reply:
@sirpy these changes did not make a trick, as the feed event modals opened from the dashboard, not through SimpleStoreDialog. The profile is still an empty object.